### PR TITLE
GUI_Resizing

### PR DESCRIPTION
**Pull request by Drayz64**
Resizing the GUI depending on the currently showing card

### CODE REVIEW

**Comment by Drayz64** [comment]
There's a slight resizing problem with the chart view card - the window is far too compact, but this is the only way I was able to get the window to resize for the charts.

I think if at the end of this sprint/before start of sprint 3 we reworked the charts so they are just cards part of the GUIManager's ViewManager there wouldn't be any problems.

File: src/main/java/tempest/ui/MyCardLayout.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+public class MyCardLayout extends CardLayout {
+
+    @Override
+    public Dimension preferredLayoutSize(Container parent) {
+        Component current = findCurrentComponent(parent);
+
+        if (current.getName() != null) {
+            String name = current.getName();
+
+            //System.out.println("Name : " + name);
```
Comment (Drayz64) [inline src/main/java/tempest/ui/MyCardLayout.java:19]: I left these prints commented out as they'll be helpful for later if we do rework the charts

File: src/main/java/tempest/ui/MyCardLayout.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package tempest.ui;
+
+import tempest.ui.components.charts.ChartTypes;
+import tempest.ui.pages.PageNames;
+
+import java.awt.*;
+
+//Coded with help of: https://stackoverflow.com/questions/8277834/how-to-set-a-jframe-size-to-fit-the-cardlayout-displayed-jpanel
+
+public class MyCardLayout extends CardLayout {
```
Comment (bridges-wood) [inline src/main/java/tempest/ui/MyCardLayout.java:10]: Can we call this FittedCardLayout?

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Just the rename, and then we're 💯

**Review by bridges-wood — APPROVED** [review]

**Pull request merged**